Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MarkdownTextBlock/MarkdownTextBlockPage.xaml.cs

```diff
@@ -79,5 +84,15 @@ private void SetInitalText(string text)
                 await Launcher.LaunchUriAsync(new Uri(e.Link));
             }
         }
+
+        // Custom Code Block Renderer
+        private void MarkdownText_CodeBlockResolving(object sender, CodeBlockResolvingEventArgs e)
+        {
+            if (e.CodeLanguage == "CUSTOM")
+            {
+                e.Handled = true;
+                e.InlineCollection.Add(new Run { Foreground = new SolidColorBrush(Colors.Red), Text = e.Text, FontWeight = FontWeights.Bold });
```

Comment:
Should this example do something with the text like look for a keyword and only make that part red?  It might be a more useful example that way.

Reply:
I don't really want to delve into Regex for this one, it is a simple sample after all.